src/drivers/v4l2/v4l2imager.cpp

```diff
@@ -176,6 +176,11 @@ void V4L2Imager::setControl(const Control &setting)
     Control new_value = setting;
     new_value.value = find_if( d->resolutions.begin(), d->resolutions.end(), [&](const V4L2Formats::Resolution::ptr &r){ return *r == *current; } ) - d->resolutions.begin();
     emit changed(new_value);
+    //TESTING ###############
```

Comment:
I guess this was meant to be removed before the merge?
Can you check please?

Reply:
You're right, now it's removed.